core/storage/base_model/gae_models.py

```diff
@@ -115,32 +117,34 @@ def get_multi(cls, entity_ids, include_deleted=False):
 
     @classmethod
     def put_multi(cls, entities):
-        """Stores sequence of entities.
+        """Stores the given ndb.Model instances.
 
         Args:
-            entities: sequence.
+            entities: list(ndb.Model).
 
-        Return:
-            list with the stored keys.
+        Returns:
+            list(ndb.Key). Each Key is referencing according Model instance.
+            Key contains id.
         """
         return ndb.put_multi(entities)
 
     def delete(self):
+        """Deletes this instance."""
         super(BaseModel, self).key.delete()
 
     @classmethod
-    def get_all(cls, include_deleted_entities=False):
+    def get_all(cls, include_deleted=False):
         """Gets iterable of all entities of this class.
 
         Args:
-            include_deleted_entities: bool. If is True then entities that have
-            been marked deleted are returned as well. Default False.
+            include_deleted: bool. If is True then entities that have
```

Comment:
bool. If True, then entities that have been marked deleted are returned as well. Defaults to False.